test/spec/ol/format/esrijsonformat.test.js

```diff
@@ -304,15 +304,15 @@ describe('ol.format.EsriJSON', function() {
               -10579560.16462974, 4716567.373073828]))
             .to.be(true);
 
-        var last = result[305];
+        var last = result[8];
```

Comment:
It's still the last feature, I just made the file smaller because it was very slow parsing in IE